dora/shaded/client/pom.xml

```diff
@@ -345,15 +345,8 @@
               <goal>copy</goal>
             </goals>
             <configuration>
-<<<<<<< HEAD:dora/shaded/client/pom.xml
-              <sourceFile>target/alluxio-shaded-client-295-SNAPSHOT.jar</sourceFile>
-              <destinationFile>../../../client/alluxio-295-SNAPSHOT-client.jar</destinationFile>
-||||||| parent of ca81323975 (Build and symlink to shaded client jar within client/build/):shaded/client/pom.xml
-              <sourceFile>${basedir}/target/${project.artifactId}-${project.version}.jar</sourceFile>
-              <destinationFile>${project.parent.parent.basedir}/client/alluxio-${project.version}-client.jar</destinationFile>
-=======
-              <sourceFile>${basedir}/target/${project.artifactId}-${project.version}.jar</sourceFile>
-              <destinationFile>${project.parent.parent.basedir}/client/build/alluxio-${project.version}-hadoop2-client.jar</destinationFile>
+              <sourceFile>target/alluxio-shaded-client-${project.version}.jar</sourceFile>
+              <destinationFile>../../../client/build/alluxio-${project.version}-hadoop2-client.jar</destinationFile>
```

Comment:
any reason we don't use `${project.parent.parent.parent.basedir}` syntax here?

Reply:
i copied this somewhere from existing dora code, all i know is this works